apps/election-manager/src/utils/semsTallies.test.ts

```diff
@@ -649,3 +650,128 @@ describe('convertSEMSFileToExternalTally', () => {
     }
   })
 })
+
+describe('parseSEMSFileAndValidateForElection', () => {
+  it('returns error for a bad precinct id', () => {
+    const csvRowRaw =
+      '"10","not-a-precinct-id","750000015","Ballot Measure 1","0","NP","1","Times Over Voted","0","NP","0",'
+    expect(
+      parseSEMSFileAndValidateForElection(
+        csvRowRaw,
+        electionWithMsEitherNeither
+      )
+    ).toEqual([
+      'Precinct ID not-a-precinct-id is not found in the election definition.',
+    ])
+  })
+
+  it('returns error for a bad precinct id', () => {
```

Comment:
Should this be `contest id`?

```suggestion
  it('returns error for a bad contest id', () => {
```